src/main/java/edu/kis/powp/jobs2d/drivers/adapter/MyDriver.java

```diff
@@ -4,15 +4,19 @@
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
 
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class MyDriver implements Job2dDriver {
+
+	private final DrawPanelController drawerController;
 	private int startX = 0, startY = 0;
 
-	public MyAdapter() {
-		super();
+	public MyDriver() {
+//		super();
```

Comment:
In revision repo it is safe to delete code instead of using a comment. 
super() does not need to be removed (for explicit coding)